src/main/proto/user-service.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package sjchat.users;
+
+option java_multiple_files = true;
+option java_package = "sjchat.users";
+option java_outer_classname = "UserServiceProto";
+
+service UserService {
+    rpc createUser (UserDataRequest) returns (UserResponse);
```

Comment:
Suggestion: use a separate request/response type for any RPC, named `RpcNameRequest / RpcNameResponse`. This means when changing one such type, not all needs to be changed.
For example, it is not unreasonable that the `createUser` call might at some point need to return additional data compared with the `getUser` call. In this situation.